packages/pwa-kit-runtime/src/utils/ssr-namespace-paths.js

```diff
@@ -5,39 +5,32 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
+import {getConfig} from './ssr-config'
+
 /**
  * This file defines the /mobify paths used to set up our Express endpoints.
  *
  * If a namespace for the /mobify paths is defined, the methods in here will return the
  * namespaced path. ie. /namespace/mobify/...
  */
 
+// The MOBIFY_PATH is defined separately in preparation for the future eventual removal or
+// replacement of the 'mobify' part of these paths
 const MOBIFY_PATH = '/mobify'
 const PROXY_PATH_BASE = `${MOBIFY_PATH}/proxy`
 const BUNDLE_PATH_BASE = `${MOBIFY_PATH}/bundle`
 const CACHING_PATH_BASE = `${MOBIFY_PATH}/caching`
 const HEALTHCHECK_PATH = `${MOBIFY_PATH}/ping`
 const SLAS_PRIVATE_CLIENT_PROXY_PATH = `${MOBIFY_PATH}/slas/private`
 
-/**
- * @private
- */
-const _getNamespace = () => {
-    // TODO - namespaces for /mobify path will be implemented at a later date.
-    // Returns an empty string for now.
-    // Below is an example of what this implementation might look like.
-    /*
-        let {namespace = ""} = getConfig()
-        namespace = typeof namespace === 'function' ? namespace() : namespace
-        return namespace
-    */
-    return ''
+// TODO - Allow projects to define this function?
+export const getEnvBasePath = () => {
+    const config = getConfig()
+    return config?.envBasePath ? config.envBasePath : ''
 }
 
-export const ssrNamespace = _getNamespace()
-
-export const proxyBasePath = `${ssrNamespace}${PROXY_PATH_BASE}`
-export const bundleBasePath = `${ssrNamespace}${BUNDLE_PATH_BASE}`
-export const cachingBasePath = `${ssrNamespace}${CACHING_PATH_BASE}`
-export const healthCheckPath = `${ssrNamespace}${HEALTHCHECK_PATH}`
-export const slasPrivateProxyPath = `${ssrNamespace}${SLAS_PRIVATE_CLIENT_PROXY_PATH}`
+export const proxyBasePath = () => `${getEnvBasePath()}${PROXY_PATH_BASE}`
+export const bundleBasePath = () => `${getEnvBasePath()}${BUNDLE_PATH_BASE}`
+export const cachingBasePath = () => `${getEnvBasePath()}${CACHING_PATH_BASE}`
+export const healthCheckPath = () => `${getEnvBasePath()}${HEALTHCHECK_PATH}`
+export const slasPrivateProxyPath = () => `${getEnvBasePath()}${SLAS_PRIVATE_CLIENT_PROXY_PATH}`
```

Comment:
These were converted into functions as the path is resolved at runtime after we are able to read the envBasePath from config.

These properties may need to be renamed but just converting them to functions may also be a breaking change =/

Reply:
For now, I will create new methods (ie. getProxyPath or getBundlePath) but I might have to keep the old strings around as deprecated properties until we can properly clean them up.